lightning/src/routing/gossip.rs

```diff
@@ -1203,6 +1225,10 @@ impl<L: Deref> ReadableArgs<L> for NetworkGraph<L> where L::Target: Logger {
 			channels: RwLock::new(channels),
 			nodes: RwLock::new(nodes),
 			last_rapid_gossip_sync_timestamp: Mutex::new(last_rapid_gossip_sync_timestamp),
+			#[cfg(feature = "std")]
+			removed_nodes: Mutex::new(HashMap::new()),
+			#[cfg(feature = "std")]
+			removed_channels: Mutex::new(HashMap::new()),
```

Comment:
Should we write these out as `Duration` like we do in the scorer? Related, see other comment about using `Time` but putting a `Duration` in the `HashMap`.